utils/docker/Dockerfile.centos.7

```diff
@@ -33,8 +33,8 @@ RUN set -e;                                                      \
     if [ -n "$REPO_FILE_URL" ]; then                             \
         dnf --quiet config-manager --disable epel;               \
     fi;                                                          \
-    dnf -y update epel-release;                                  \
-    dnf -y clean all
+    dnf update epel-release;                                     \
```

Comment:
I don't understand why we need any changes in this directory at all.

Reply:
https://daosio.atlassian.net/browse/DAOS-10684

I tripped over that when I started working on the PR.

The sudo setup for the existing dockerfiles was also incorrect.

Reply:
While working on this PR, one of the public mirrors for OpenSUSE went offline for a few hours and I had to fix the Dockerfiles to be able to install curl to get the Artifactory repositories installed with out depending on the public mirrors.